user/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
```

Comment:
There shouldn't be a "user" folder in your structure anymore. We may need to discuss this in person.